README.md

```diff
@@ -131,6 +132,11 @@ Also it has only 20 (instead of 38) parallel environments to improve sample effi
 
 * [Fetch Environments (sparse rewards) w/ Action Repeat](alf/examples/sac_actrepeat_fetch/sac_actrepeat_fetch.gin). We are able to achieve even better performance than reported by DDPG + Hindsight Experience Replay in some cases simply by using SAC + Action Repeat with length 3 timesteps. See [this note](docs/notes/fetch_env_sac_actrepeat_notes.rst) to view learning curves, videos, and more details.
 
+### HER
+* [Goal Conditioned Target Navigation](alf/examples/her_target_navigation_states.gin). Need to install SocialRobot.
+
+<img src="alf/examples/her_target_navigation_states.png" width="500" height="200" alt="sac-fetchslide-training-curve"/>
```

Comment:
The visual effect is very different from others. Others have animated gif. If you want to add example here, try to make it consistent with other examples.